bridge/engine/AudioForwarderReceiveJob.cpp

```diff
@@ -132,23 +132,45 @@ void AudioForwarderReceiveJob::run()
     const auto rtpHeaderExtensions = rtpHeader->getExtensionHeader();
     if (rtpHeaderExtensions)
     {
-        int32_t audioLevel = -1;
+        auto c9infoExtId = _ssrcContext._rtpMap._c9infoExtId.valueOr(0);
+        auto audioLevelExtId = _ssrcContext._rtpMap._audioLevelExtId.valueOr(0);
+
+        utils::Optional<uint8_t> audioLevel;
+        utils::Optional<bool> isPtt;
 
         for (const auto& rtpHeaderExtension : rtpHeaderExtensions->extensions())
         {
-            if (!_ssrcContext._rtpMap._audioLevelExtId.isSet() ||
-                rtpHeaderExtension.getId() != _ssrcContext._rtpMap._audioLevelExtId.get())
+            if (0 != c9infoExtId && rtpHeaderExtension.getId() == c9infoExtId)
             {
-                continue;
+                isPtt.set(rtpHeaderExtension.data[3] & 0x80);
+            }
+            else if (0 != audioLevelExtId && rtpHeaderExtension.getId() == audioLevelExtId)
+            {
+                audioLevel.set(rtpHeaderExtension.data[0] & 0x7F);
             }
-
-            audioLevel = rtpHeaderExtension.data[0] & 0x7F;
-            break;
         }
 
-        _activeMediaList.onNewAudioLevel(_sender->getEndpointIdHash(), audioLevel);
+        bool silence = false;
+        if (isPtt.isSet())
+        {
+            silence = !isPtt.get();
+            _activeMediaList.onNewPtt(_sender->getEndpointIdHash(), isPtt.get());
+            _activeMediaList.onNewAudioLevel(_sender->getEndpointIdHash(), silence ? 127 : 0);
+        }
+        else
+        {
+            if (audioLevel.isSet())
```

Comment:
should be 
else if(audioLevel.isSet()) 
But now ptt has precedence over audio level since ptt.isSet is chcked first.

Reply:
If we have audio level we should report it regardless of ptt.
If we have ptt but not audiolevel, we can fake audiolevel as you do

Reply:
My guess, is that if we have both, ptt is going to be 1 along with fairly high audio level. 
If ptt is 1 and audio level low, I prefer if someone else that has audio is heard.

Reply:
Ok, reversing precendence, now we have:
```
        if (audioLevel.isSet())
        {
            silence = audioLevel.get() > _silenceThresholdLevel;
        }
        else if (isPtt.isSet())
        {
            silence = !isPtt.get();
            audioLevel.set(silence ? 127 : 0);
            _activeMediaList.onNewPtt(_sender->getEndpointIdHash(), isPtt.get());
        }
```